src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/TextBox.cs

```diff
@@ -76,6 +76,9 @@ static TextBox()
              new CoerceValueCallback(CoerceHorizontalScrollBarVisibility)));
 
             ControlsTraceLogger.AddControl(TelemetryControls.TextBox);
+        
+            // CommandHelpers.RegisterCommandHandler(typeof(TextBox), TextBox.ClearCommand, OnClearCommand);
```

Comment:
Should remove?

Reply:
Yes it can be removed. Will go ahead and do the same.